src/services/infra/InfraService.ts

```diff
@@ -229,6 +260,159 @@ export default class InfraService {
     return ok(dnsRecord)
   }
 
+  launchSiteFromCms({
```

Comment:
out of curiosity, why prefer object notation over the positional one

Reply:
Personally tend to use object notation for more than 2 args. Even though it is more verbose, it makes the code alot clearer. Too many instances for myself when I debug only to find out the issue was the ordering of the args, would advocate for this esp since we still have js code :sadge:

Reply:
can. as a side note, usually people do main args as positional and options as object. this is because you want consumers to have an easier time writing the method (object very annoying) but when there are options, be precise about the options